WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -201,9 +201,8 @@
     context 'when signed in as an admin' do
       sign_in { FactoryBot.create :admin }
 
-      it 'creates a new competition' do
+      it "creates a new competition with organizers and expect them to receive a notification email" do
         post :create, params: { competition: { name: "FatBoyXPC 2015" } }
-        expect(response).to redirect_to edit_competition_path("FatBoyXPC2015")
```

Comment:
I will revert these two changes (description and missing line) after I got some reviews, they're only some leftovers of some tests.